.tekton/push.yaml

```diff
@@ -213,50 +134,44 @@ spec:
 
                 .tekton/scripts/build-acceptable-bundles.sh "$@"
 
-                echo -n "$DATA_BUNDLE_REPO" > "$(results.DATA_BUNDLE_REPO.path)"
-                echo -n "$DATA_BUNDLE_TAG" > "$(results.DATA_BUNDLE_TAG.path)"
+                DATA_BUNDLE_KONFLUX_CI=quay.io/konflux-ci/tekton-catalog/data-acceptable-bundles
+                skopeo copy \
+                  "docker://$DATA_BUNDLE_REPO:$DATA_BUNDLE_TAG" \
+                  "docker://$DATA_BUNDLE_KONFLUX_CI:$DATA_BUNDLE_TAG"
+
+                skopeo copy \
+                  "docker://$DATA_BUNDLE_REPO:$DATA_BUNDLE_TAG" \
+                  "docker://$DATA_BUNDLE_KONFLUX_CI:latest"
               args:
-                - $(workspaces.artifacts.path)/task-bundle-list
-                - $(workspaces.artifacts.path)/pipeline-bundle-list
-                - $(workspaces.artifacts-redhat-appstudio.path)/task-bundle-list
-                - $(workspaces.artifacts-redhat-appstudio.path)/pipeline-bundle-list
+                - $(workspaces.source.path)/task-bundle-list-konflux-ci
+                - $(workspaces.source.path)/pipeline-bundle-list-konflux-ci
+                - $(workspaces.source.path)/task-bundle-list-appstudio
+                - $(workspaces.source.path)/pipeline-bundle-list-appstudio
               volumeMounts:
                 - mountPath: /root/.docker/config.json
                   subPath: .dockerconfigjson
                   name: quay-secret
```

Comment:
I don't think the task will be able to push both to quay.io/konflux-ci and to quay.io/redhat-appstudio-tekton-catalog with same dockerconfig